Source/Blazorise/Components/Validation/ValidationRule.cs

```diff
@@ -182,5 +184,36 @@ public static void IsFileSelected( ValidatorEventArgs e )
         }
     }
 
+    /// <summary>
+    /// Checks if the date is selected.
+    /// </summary>
+    /// <typeparam name="TValue">Data-type used be binded by the Date property.</typeparam>
+    /// <param name="e"></param>
+    public static void IsDateSelected<TValue>( ValidatorEventArgs e )
+    {
+        var dates = e.Value as IEnumerable<TValue>;
+
+        if ( dates is not null )
```

Comment:
What happens if eventually null?

In the `AreDatesSelected` the `ValidationStatus.Error` is explicitly set.